From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Thu, 15 Sep 2011 19:46:05 -0400
Subject: drivers/net: Add moduleparam.h to drivers as required.
Patch-mainline: v3.2-rc1
Git-commit: 6eb07caf1ac5723720caea2ee93cd11b7058a0aa (partial)
References: bnc#786036 FATE#314304

These files were using moduleparam infrastructure, but were not
including anything for it -- which is fine when module.h is being
implicitly included in all files, but that is going away.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/mellanox/mlx4/en_tx.c |    1 +
 1 file changed, 1 insertion(+)
--- a/drivers/net/ethernet/mellanox/mlx4/en_tx.c
+++ b/drivers/net/ethernet/mellanox/mlx4/en_tx.c
@@ -39,6 +39,7 @@
 #include <linux/if_vlan.h>
 #include <linux/vmalloc.h>
 #include <linux/tcp.h>
+#include <linux/moduleparam.h>
 
 #include "mlx4_en.h"
 
